### PR TITLE
Starboard Embed image added and GuildConfig NPE solved

### PR DESCRIPTION
**Pull request by HGotthard**
## Requirements for Contributing

* [x] The changed and/or added Code should be documented and understandable.
* [x] The changed and/or added Code should follow the Code Conventions.

## Type of the Pull Request
* [x] Bugfix
* [ ] Feature change
* [ ] Documentation change
* [ ] Something different

## Description of the Changes
- Adds attachements to starboard embeds to show images.
- Replaces json to list function in Settings to remove warnings.
- Moves GuildConfig first creation to onGuildJoin to prevent NPE.

### CODE REVIEW

File: src/main/java/de/progen_bot/util/Settings.java [hunk 1]
```diff
@@ -36,13 +36,12 @@ private Settings() {
...
     //#region Bot
-    public static final List<String> BOT_OWNERS =   Collections.unmodifiableList(GSON.fromJson(get(BOT_KEY, "owners").getAsJsonArray(), List.class));
+    public static final List<String> BOT_OWNERS =   Arrays.asList(GSON.fromJson(get(BOT_KEY, "owners").getAsJsonArray(), String[].class));
     public static final String PREFIX           =   get(BOT_KEY, "prefix").getAsString();
     public static final String TOKEN            =   get(BOT_KEY, "token").getAsString();
     public static final String API_PORT         =   String.valueOf(get(BOT_KEY, "apiPort").getAsInt());
 
-    @SuppressWarnings("unchecked")
-    public static final List<String> MUSIC      =   Collections.unmodifiableList(GSON.fromJson(get(BOT_KEY, "music").getAsJsonArray(), List.class));
+    public static final List<String> MUSIC      =   Arrays.asList(GSON.fromJson(get(BOT_KEY, "music").getAsJsonArray(), String[].class));
```
Comment (byNoobiYT) [inline src/main/java/de/progen_bot/util/Settings.java:44]: Lists should be unmodifiable.

File: src/main/java/de/progen_bot/listeners/StarBoardListener.java [hunk 2]
```diff
@@ -71,7 +71,10 @@ private void printResult(String guildId, StarBoard starBoard, int count) {
                             .setColor(Color.YELLOW)
                             .setAuthor(message.getAuthor().getAsTag(), null, message.getAuthor().getAvatarUrl())
                             .setTitle("Jump to message", message.getJumpUrl())
-                            .setDescription(message.getContentDisplay()).build();
+                            .setDescription(message.getContentDisplay())
+                            .setImage(message.getAttachments().isEmpty() ? null
+                                    : message.getAttachments().get(0).getUrl())
```
Comment (byNoobiYT) [inline src/main/java/de/progen_bot/listeners/StarBoardListener.java:76]: Attachment can be also .txt files, maybe check file ending?

**Review by byNoobiYT — CHANGES_REQUESTED** [review]

File: src/main/java/de/progen_bot/util/Settings.java (hunk 1 above)
Comment (HGotthard) [reply src/main/java/de/progen_bot/util/Settings.java]: Yeah true. Thought its done with final but thats only for initialization.

**Review by byNoobiYT — APPROVED** [review]
Seems fine

**Pull request merged**